### ISSUE TITLE
ImportError: 'simps' from scipy.integrate

### ISSUE THREAD

**Issue by ashutoshjha3103** [cosanlab/py-feat#242]
After `pip install py-feat`, while importing Detector class, this error pops up under feat/utils/stats.py

ImportError: cannot import name 'simps' from 'scipy.integrate'

This is due to a version conflict, where this function is now called 'simpson' and not simps

Issue discussed here: https://github.com/XanaduAI/strawberryfields/issues/750

**Comment by avshyz** [comment]
@ashutoshjha3103 - did you find any workaround?

**Comment by ashutoshjha3103** [comment]
@avshyz yes, for now I use the scipy==1.13 (<1.14) as suggested on the link I gave in the description.

**Comment by ljchang** [comment]
Now imports scipy.integrate.simpson (the renamed simps). Fixed on the `v0.7-dev` branch (ships in v0.7.0). Please reopen if you still hit this on `v0.7-dev`.
